docs/pages/operations-guide.md

```diff
@@ -288,9 +288,9 @@ Specifying that pod join the `dcos` overlay network has the following indirect e
 
 ## Secrets
 
-Enterprise DC/OS provides Secrets feature.  Secrets enable secure storage and transportation and fine-grained access control to store sensitive data such as database passwords, private keys, API tokens. More information about Secrets is available at `docs.mesosphere.com`.
+Enterprise DC/OS provides a secrets store to control storage and provide access to sensitive data such as database passwords, private keys, and API tokens. [Learn more about the secrets store](https://docs.mesosphere.com/1.9/security/secrets/).
```

Comment:
"Enterprise DC/OS provides a secrets store to control storage ...." How about:

"Enterprise DC/OS provides a Secrets store to enable access to sensitive data such as database passwords, private keys, and API tokens. DC/OS manages secure transportation of secret data, access control and authorization, and secure storage of secret content.
[Learn more about the secrets store](https://docs.mesosphere.com/1.9/security/secrets/).

Note: Dcos-commons SDK supports Secrets in Enterprise DC/OS 1.10 onwards  (not in Enterprise DC/OS 1.9)
"